x/twasm/contract/incoming_msgs.go

```diff
@@ -279,6 +281,22 @@ type ConsensusParamsUpdate struct {
 	Evidence *EvidenceParams `json:"evidence,omitempty"`
 }
 
+// Delegate.
+// See
+type Delegate struct {
+	Denom      string `json:"denom"`
+	Amount     string `json:"amount"`
+	SenderAddr string `json:"sender"`
```

Comment:
Sender is a bit ambiguous as this comes from a contract. Would `stakerAddr` or `delegatorAddr` makes sense?